### ISSUE TITLE
SmartCity-Accessibility Server Side Long Tests #22 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#357]
Build 'SmartCity-Accessibility Server Side Long Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.84 KB...]
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 20, 2017
JxMaps license valid.
Running smartcity.accessibility.services.LoginServiceTest
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 12582912 bytes for committing reserved memory.
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00000000fd500000, 12582912, 0) failed; error='Cannot allocate memory' (errno=12)
# An error report file with more information is saved as:
# /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/hs_err_pid32084.log

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Skipping SmartCity-Accessibility
[INFO] This project has been banned from the build due to previous failures.
[INFO] ------------------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:45 min
[INFO] Finished at: 2017-05-20T16:41:56+00:00
[INFO] Final Memory: 39M/202M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/jacoco.exec -jar /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefirebooter398713615960402189.jar /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefire1574061614875939501tmp /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefire_04579947837311782417tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Ariel Kolikant] 0387de6f044df0fd17f8f7e382de07ee368cd9cc - hardcoded locations from modiin, will soon add usage of parameters for

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Long%20Tests/22/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
